app/public/css/styles.css

```diff
@@ -3775,6 +3775,23 @@ input:checked + .slider:before {
   margin-bottom: 32px;
 }
 
+/* MULTI ASSETS SCREEN: START */
```

Comment:
What do you think about those "start/end" comments? To make the "end" more explicit